gradle.properties

```diff
@@ -1,5 +1,4 @@
 # Kotlin
-kotlin.version=1.8.0
```

Comment:
after some investigation this is never used. we define our Kotlin version in `buildSrc/src/config` so it's fine to delete it.